keps/sig-storage/2485-read-write-once-pod-pv-access-mode/README.md

```diff
@@ -847,12 +981,24 @@ Longer term, we may want to require automated upgrade/rollback tests, but we
 are missing a bunch of machinery and tooling and can't do that now.
 -->
 
+For alpha, manual tests were performed to:
+
+- Unsuccessfully create workloads using ReadWriteOncePod PVCs prior to upgrade
```

Comment:
It is not possible to create a ReadWriteOncePod PVCs prior to upgrade, right?

Reply:
Correct, the API server will return an error saying it doesn't know about this access mode.